src/Abc.Zebus/Directory/SubscriptionDefinition.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Abc.Zebus.Routing;
+
+namespace Abc.Zebus.Directory
+{
+    public class SubscriptionDefinition
+    {
+        public List<SubscriptionDefinitionPart> Parts { get; }
```

Comment:
`IReadOnlyList<SubscriptionDefinitionPart>` or `SubscriptionDefinitionPart[]` would be better for a public API

Note to self: check later if this class needs to be public

Reply:
How could it not be public it's returned as part of the public API on `Subscription`

Reply:
That was just a note to self 😉

Reply:
The comment on the property type is still valid though